src/Compilers/Core/Portable/Generated/Operations.Generated.cs

```diff
@@ -3955,6 +3955,14 @@ public interface ICollectionExpressionOperation : IOperation
         /// </summary>
         IMethodSymbol? ConstructMethod { get; }
         /// <summary>
+        /// Arguments passed to a <c>with(...)</c> element on the collection expression, if any.
```

Comment:
>Arguments passed to a <c>with(...)</c> element on the collection expression, if any. [](http://example.com/codeflow?start=12&length=84)

This comment looks vary vague to me. In success cases, these are actually the arguments passed to ```ConstructMethod```, they will include default values, possibly param arrays, etc. Reading the current description, one might find it unexpected to run into arguments that aren't actually present in ```with(...)``` syntax.